doc/install.rst

```diff
@@ -83,7 +83,17 @@ Test
 ====
 
 If you are developing for the RAMP package, you will be interested about
-testing your new feature. You can run the test using ``pytest`` from the root
+testing your new feature. First you need to `set up the database <https://paris-saclay-cds.github.io/ramp-docs/stable/setup_server.html#set-up-the-ramp-database>`_ and create the test database::
+
+    createdb --owner=<the owner you specified> databoard_test
```

Comment:
Here you will not have any choice. The config that we are testing thing require `mrramp` as password and user.